### ISSUE TITLE
Has any plan  Importing from UI Libraries  support Quasar

### ISSUE THREAD

**Issue by fyeeme** [antfu/unplugin-vue-components#91]
after view the doc, i found this module support 

Ant Design Vue
Element Plus

Has any plan  Importing from UI Libraries  support **Quasar**

**Comment by antfu** [comment]
Feel free to create a PR. Thanks

**Comment by fyeeme** [comment]
@vitorarantes  Just wanna to know have ever test  the resolver and can offer a codebox?  I create the issue cause that I'v try some other ideas like [https://github.com/ekoeryanto/vite-quasar/tree/master/scripts](https://github.com/ekoeryanto/vite-quasar/tree/master/scripts)
it really works  except A bit wordy! also I create a PR for it ,but still not good enough

but as  I tested on this  reslover, seam not work, not register quasar register fn, not register style , cannot use plugin

so have I missed something?

**Comment by fyeeme** [comment]
@antfu  And also I think this issue should be reopened!

**Comment by vitorarantes** [comment]
@fyeeme did you import Quasar in your main.js / app.js ?

```
import { Quasar } from 'quasar'
import 'quasar/dist/quasar.css'

app.use(Quasar, {
  config: { //... },
  plugins: { //... }
})
```
